package.json

```diff
@@ -11,6 +11,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.18.2"
+    "express": "^4.18.2",
+    "nodemon": "^2.0.20"
```

Comment:
why  you need that here? I should be under dev dependency or not needed at all
You should have nodemon install globally on your pc.